### DISCUSSION TITLE
"Tower" like Theme for JsTree?

### DISCUSSION THREAD

**Discussion by lexterror** [vakata/jstree#2510] [Show and tell]
I've always had issues using js/css/html "trees/search trees." I've always felt like they're not very "Symmetry/Layout" friendly. I've created a basic modification to your program "JsTree." You can let me know what you think. Maybe my modification can be improved or enhanced if found useful.

The demo can be found here:  http://electrolex.myartsonline.com/

Thank you!

**Comment by vakata** [comment]
Thank you for sharing your work - looks great!
